docs/ORTModule_Training_Guidelines.md

```diff
@@ -495,3 +495,31 @@ for epoch in range(start_epoch, n_epochs):
 ```
 
 Check [LoadBalancingDistributedBatchSampler implementation](../orttraining/orttraining/python/training/utils/data/sampler.py) for more details.
+
+## 8 Using ORTPipelineModule for Deepspeed Pipeline Parallel
+
+You can use `ORTPipelineModule` to support Deepspeed Pipeline Parallelism. Here's how you can integrate it into your pipeline:
+
+```python
+from onnxruntime.training.ortmodule import DebugOptions
+from onnxruntime.training.ortmodule.experimental.pipe import ORTPipelineModule
+
+# Create a debug configuration if needed
+# Since we're exporting multiple graphs here, this will generate multiple graphs with their index added as a prefix to differentiate them.
+
+debug_options = DebugOptions(save_onnx=True, log_level=LogLevel.VERBOSE, onnx_prefix="model_name")
+
+# Keep your deepspeed script the same and use ORTPipelineModule instead of PipelineModule
+# Initialize the ORTPipelineModule
+pipeline_module = ORTPipelineModule(
+    layers,
```

Comment:
What is `layers`? Please ensure each code example is runnable by just copying and pasting it to Python Interpreter.

Reply:
We can't have a full code example as it depends on deepspeed, we can add this to the onnxruntime-training-examples repository.

Those who already have a DeepSpeed example will have the layers object constructed. 
We're only showing the changes needed to run in ORT.

Reply:
If so, add a relative path to your test file and instructions to run that test file here. After reading this example, user must be able to run `ORTPipelineModule`.